src/App.tsx

```diff
@@ -1,14 +1,56 @@
 /* eslint-disable max-len */
-import React from 'react';
+import React, { useState, useMemo, useEffect } from 'react';
 import 'bulma/css/bulma.css';
 import '@fortawesome/fontawesome-free/css/all.css';
 
 import { TodoList } from './components/TodoList';
 import { TodoFilter } from './components/TodoFilter';
 import { TodoModal } from './components/TodoModal';
 import { Loader } from './components/Loader';
+import { getTodos } from './api';
+import { Todo } from './types/Todo';
+import { Status } from './types/Status';
+
+function getPreparedTodos(todoList: Todo[], select: Status, query: string) {
+  const visibleTodos = [...todoList];
+
+  return visibleTodos
```

Comment:
```suggestion
  return todoList
```
filter doesn't mutate the array, so this copy is redundant here